src/server/services/bezwaren/bezwaren.ts

```diff
@@ -17,7 +17,7 @@ import {
 import { isRecentNotification } from '../../../universal/helpers/utils';
 import { AuthProfileAndToken, generateFullApiUrlBFF } from '../../helpers/app';
 import { requestData } from '../../helpers/source-api-request';
-import { decryptAndValidate } from '../shared/decrypt-route-param';
+import { decryptEncryptedRouteParamAndValidateSessionID } from '../shared/decrypt-route-param';
```

Comment:
als je iets decrypt dan weet je al dat het encrypted is dus misschien
```suggestion
import { decryptRouteParamAndValidateSessionID } from '../shared/decrypt-route-param';
```
?